share/github-backup-utils/ghe-backup-alambic-cluster-ng

```diff
@@ -85,6 +85,7 @@ bm_end "$(basename $0) - Processing routes"
 # rsync all the repositories
 bm_start "$(basename $0) - Storage object sync"
 for file_list in $tempdir/*.rsync; do
+  [[ -f "$file_list" ]] || continue
```

Comment:
Nit: we'd prefer to only [use `[[` for advanced bash functionality](https://github.com/github/backup-utils/blob/master/STYLEGUIDE.md#scripts-may-use--for-advanced-bash-features).

I also think it may be a good idea to only enter the loop if there is something to sync. I believe we should be able to guard against this by checking that `$routes_list` isn't empty. If it is, then there won't be any `$tempdir/*.rsync` files so we don't need to enter the loop.

Reply:
@lildude Thanks, that's the feedback I was looking for 😄  :+1:

Reply:
I've got a local fix for this from some recent testing. Mind if I ~~trample all over~~ update your branch @donal?

Reply:
@lildude Thank you, please do! 🐘